src/yggdrasil/config/config.go

```diff
@@ -5,6 +5,7 @@ type NodeConfig struct {
 	Listen      string
 	AdminListen string
 	Peers       []string
+	PeerBoxPubs []string
```

Comment:
Internally the name `PeerBoxPubs` I guess is accurate, but it isn't an intuitive name, and someone looking at the nodeconfig without documentation would probably have no idea what this does. Would it be better to give it a more friendly name, i.e. `AllowedBoxPubs` or `AllowedPublicKeys` or so?